molecule.yml

```diff
@@ -2,6 +2,11 @@
 ansible:
   verbose:
   sudo: True
+  host_vars:
```

Comment:
Fixed in [bbb9143](https://github.com/azavea/ansible-pip/pull/2/commits/bbb91434b15d0addfbfa1bb0eb81075b86e593ec). 

Also figured out a way to check the version of pip that gets installed, wrote about it [here](https://slack-files.com/T029GD0MJ-F26QFVASX-e6721c034a)